doc/templates/GBFS-Config.md

```diff
@@ -0,0 +1,20 @@
+<!--
+  NOTE! Part of this document is generated. Make sure you edit the template, not the generated doc.
+
+   - Template directory is:  /doc/templates
+   - Generated directory is: /doc/user 
+-->
+
+Besides GTFS-RT or SIRI transit data, OTP can also fetch real-time data about vehicle rental networks
+including the number of vehicles and free parking spaces at each station. We support vehicle rental
+systems that use the GBFS standard.
+
+[GBFS](https://github.com/NABSA/gbfs) can be used for a variety of shared mobility services, with
+partial support for both v1 and v2.2 ([list of known GBFS feeds](https://github.com/NABSA/gbfs/blob/master/systems.csv)). OTP supports the following
```

Comment:
What has "partial support for both v1 and v2.2" and if we are talking about OTP, should we mention what is the current supported version?

Reply:
It's hard to read because of the link but the sentence is "GBFS can be used for a variety of shared mobility services, with partial support for both v1 and v2.2."

Reply:
I find this good enough but if you tell me your preference.

Reply:
I still don't understand what the sentence tries to communicate. What has partial support? To me it reads as it says "GBFS has partial support for v1 and v2.2." which makes no sense since they are its own versions.

Reply:
It means "OTP partially supports GBFS v1 and v2.2"

Reply:
I've rewritten the paragraphs a little. Let me know if you like it.

Reply:
Maybe
```
[GBFS](https://github.com/NABSA/gbfs) can be used for a variety of shared mobility services. OTP has a comprehensive support for v2.3 and a partial support for v1 and v2.2.
```

Reply:
I saw you had already updated it. Your change looks fine.